packages/core/src/common/vars/store-migration-version.injectable.ts

```diff
@@ -2,12 +2,11 @@
  * Copyright (c) OpenLens Authors. All rights reserved.
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
-import { applicationInformationToken } from "@k8slens/application";
 import { getInjectable } from "@ogre-tools/injectable";
 
 const storeMigrationVersionInjectable = getInjectable({
   id: "store-migration-version",
-  instantiate: (di) => di.inject(applicationInformationToken).version,
+  instantiate: () => "6.4.0",
```

Comment:
Could this remain as `di.inject(applicationInformationToken).version` unless it is not our usual versioning value. Can you easily detect when `di.inject(applicationInformationToken).version` is unusual?